src/componenets/FolderDetail.tsx

```diff
@@ -9,14 +9,44 @@ import remove from "../images/remove.svg";
 import useFetchData from "../hooks/useFetchData";
 import CardList from "./CardList";
 import NoLink from "./NoLink";
-export const UserContext = React.createContext();
 
-function FolderDetail({ folderListData, toggleModal, handleFolderName }) {
-  const [selectedFolder, setSelectedFolder] = useState({});
-  const selectedFolderData =
+type Context = (id: number) => void;
+
+interface Folder {
+  id: number;
+  created_at: string;
+  name: string;
+  user_id: number;
+  favorite: boolean;
+  link: {
+    count: number;
+  };
+}
+type selectedFolderDataType = {
```

Comment:
타입스크립트의 타입(interface, type)의 네이밍은 파스칼케이스가 컨벤션입니다~
```suggestion
type SelectedFolderDataType = {
```